src/media/dummyData.tsx

```diff
@@ -0,0 +1,98 @@
+export const needsDummy = [
```

Comment:
move this file to __tests__/mock, thsi folder is for static files :) 

Same for footer file in the /pages directory - it should be in /components/footer/Footer.tsx :)